Library/Tracking/KSRAnalytics.swift

```diff
@@ -827,6 +833,7 @@ public final class KSRAnalytics {
    - parameter page: The `PageContext` representing the specific area the UI is interacted in
    - parameter checkoutData: The `CheckoutPropertiesData` associated with this specific checkout instance
    - parameter project: The `Project` corresponding to the card that was clicked
+   - parameter typeContext: The context of the  project clicked.
```

Comment:
Additional information about an event that was not captured in other context properties for the `Project` tapped.

Reply:
Can you wrap Project with `` so it is highlighted when it is option clicked.